compilers/BUILD

```diff
@@ -1 +1,2 @@
 # This empty BUILD file is required to make Bazel treat this directory as a package.
+
```

Comment:
Shouldn't we remove that?

Reply:
Yep. I just did it to force the rebuild.